components/phpunit_bridge.rst

```diff
@@ -0,0 +1,129 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report trigerred errors,
+    legacy tests and usage of deprecated code.
```

Comment:
I think we should rewrote the component summary (these lines) a bit. The summary only talks about the deprecation utilities, but it provides other testing utilities as well. I think we should change it to something like:

```rst
    The PHPUnit Bridge provides testing utilities for PHPunit, including a deprecation
    helper, a helper to mock time and a bootstrap file to setup a consistent testing
    environment.
```